README.md

```diff
@@ -107,21 +101,23 @@ The following flags can be passed to `cmake`:
 
 #### Servatrice Docker container
 
-A Dockerfile is provided to run Servatrice (the Cockatrice server) using [Docker](https://www.docker.com/what-docker).<br>
+Use [Docker](https://www.docker.com/what-docker) to run Servatrice (the Cockatrice server) using the Cockatrice Dockerfile.<br>
```

Comment:
Does this only sounds weird to me?
~`Use Docker to run Servatrice using the Cockatrice Dockerfile.`

Reply:
Sounds a bit weird to me too. Maybe "You can use Docker to run your own instance of Servatrice." ?

Reply:
I tried to improve the wording a bit.